Header header

# Vicon ground truth
geometry_msgs/Vector3 vicon_position_W            # x, y, z
geometry_msgs/Vector3 vicon_velocity_W            # x_dot, y_dot, z_dot
geometry_msgs/Vector3 vicon_velocity_B            # u, v, w
geometry_msgs/Vector3 vicon_euler_angles          # phi, theta, psi
geometry_msgs/Vector3 vicon_rotation_speed_B      # p, q, r

# Camera
